launch/launch/actions/execute_process.py

```diff
@@ -190,14 +190,14 @@ def __init__(
         self.__cmd = [normalize_to_list_of_substitutions(x) for x in cmd]
         self.__name = name if name is None else normalize_to_list_of_substitutions(name)
         self.__cwd = cwd if cwd is None else normalize_to_list_of_substitutions(cwd)
-        self.__env = None  # type: Optional[List[Tuple[List[Substitution], List[Substitution]]]]
+        self.__env = None   # type: Optional[List[Tuple[List[Substitution], List[Substitution]]]]
         if env is not None:
             self.__env = []
             for key, value in env.items():
                 self.__env.append((
                     normalize_to_list_of_substitutions(key),
                     normalize_to_list_of_substitutions(value)))
-        self.__additional_env: Optional[List[Tuple[List[Substitution], List[Substitution]]]] = None
+        self.__additional_env = None
```

Comment:
You can explicitly ignore the line length constraint for the line by appending `# noqa: E501`:

```suggestion
        self.__additional_env = None  # type: Optional[List[Tuple[List[Substitution], List[Substitution]]]]  # noqa: E501
```

Reply:
Is having a noqa'd line preferable to ditching the type comment? I wasn't sure.

Reply:
Since we're not running mypy tests it's probably okay to ditch the comment. I don't have a preference.

Reply:
I'll prefer to leave it as is then.